http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/MonitorAtBoot.scala

```diff
@@ -351,6 +458,101 @@ class MonitorAtBoot[F[_]](publisherQueue: Queue[F, LeoPubsubMessage],
         )
       case x => F.raiseError(MonitorAtBootException(s"Unexpected status for runtime ${runtime.id}: ${x}", traceId))
     }
+
+  private def getLandingZoneResources(billingProfileId: String, userToken: Authorization)(implicit
```

Comment:
I don't really like duplicating this code -- can we put this method in a shared location?

Reply:
sounds good yeah